public/views/checkout.html

```diff
@@ -0,0 +1,227 @@
+                    <div class="col-md-8">
+                        <input class="form-control" type="text" name="email" placeholder="goodsamaritan@gmail.com" >
+                    </div>
+                </div>
+                <div class="form-group">
+                    <label class="col-md-4 control-label">Address</label>
+                    <div class="col-md-8">
+                        <input class="form-control" type="text" name="address" placeholder="Street" >
+                    </div>
+                </div>
+                <div class="form-group">
+                    <label class="col-md-4 control-label"></label>
+                    <div class="col-md-8">
+                        <input class="form-control" type="text" name="address-city" placeholder="City" >
+                    </div>
+                </div>
+                <div class="form-group">
+                    <label class="col-md-4 control-label"></label>
+                    <div class="col-md-8">
+                        <input class="form-control" type="text" name="address-country" placeholder="Country" >
+                    </div>
+                </div>
+                <div class="form-group">
+                    <label class="col-md-4 control-label">Email</label>
+                    <div class="col-md-8">
+                        <input class="form-control" type="text" name="email" placeholder="goodsamaritan@gmail.com" >
+                    </div>
+                </div>
+                <div class="form-group">
+                    <label class="col-md-4 control-label">Card Number</label>
+                    <div class="col-md-8">
+                        <input class="form-control" type="text" name="email" placeholder="xxxx xxxx xxxx xxxx" >
+                    </div>
+                </div>
+                <div class="form-group">
+                    <label class="col-md-4 control-label">CSV</label>
+                    <div class="col-md-8">
+                        <input class="form-control" type="text" name="email" placeholder="123" >
+                    </div>
+                </div>
+                <div class="form-group">
+                    <label class="col-md-4 control-label">Expiration Date</label>
+                    <div class="expiration-date">
+                        <div class="col-md-8">
+                            <div class="dropdown">
+                                <button class="btn btn-default dropdown-toggle" type="button" id="dropdownMenu1" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
+                                01
+                                <span class="caret"></span>
+                                </button>
+                            <ul class="dropdown-menu" aria-labelledby="dropdownMenu1">
+                                <li><a href="#">02</a></li>
+                                <li><a href="#">03</a></li>
```

Comment:
ah, I get it. yeah. these all need to be fixed and css needs to be adjusted so that it doesn't run off the bottom of the page